eip-5564.md

```diff
@@ -234,6 +234,11 @@ You can find an implementation of this standard in TBD.
 
 The funding of the stealth address wallet represents a known issue that might breach privacy. The wallet that funds the stealth address MUST NOT have any physical connection to the stealth address owner in order to fully leverage the privacy improvements.
 
+In addition, there are potential DoS vectors that are not mitigated by protocol-level pricing. Stealth transfer senders cause an externality for recipients, as parsing announcement events consumes computational resources that are not compensated with gas. Therefore, spamming announcement events can be a detriment to the user experience, as it can lead to longer parsing times. 
+We consider the incentives to carry out such an attack as low because no monetary benefit can be obtained and, in theory, nothing prevents the community to abandoning the DoS'ed contract for setting up a new one (making the attackers effors meaningless).
```

Comment:
```suggestion
We consider the incentives to carry out such an attack as low because no monetary benefit can be obtained and, in theory, nothing prevents the community to abandoning the DoS'ed contract for setting up a new one (making the attackers efforts meaningless).
```